not sure what to do with missing crt functions in uwp:
I will not develop further.

https://docs.microsoft.com/en-us/cpp/cppcx/crt-functions-not-supported-in-universal-windows-platform-apps?view=msvc-160

the error was:

```text

\HeteroVector.cc" "D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Vectors\HeteroView.cc" "D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Vectors\HeteroPtrView.cc" "D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\ThreadGranularity.cc" "D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc"
         HeteroVector.cc
         HeteroView.cc
         HeteroPtrView.cc
         ThreadGranularity.cc
         DateTime.cc
     4>D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\include\DataFrame/Utils/FixedSizeString.h(312,1): warning C4305: 'initializing': truncation from 'unsigned __int64' to 'hmdf::VirtualString::size_type' (compiling source file D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc) [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\include\DataFrame/Utils/FixedSizeString.h(312,29): warning C4309: 'initializing': truncation of constant value (compiling source file D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc) [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\include\DataFrame/Utils/FixedSizeString.h(315,39): warning C4244: '=': conversion from 'unsigned __int64' to 'hmdf::VirtualString::size_type', possible loss of data (compiling source file D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc) [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc(129,5): error C3861: '_tzset': identifier not found [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc(972,9): error C3861: '_tzset': identifier not found [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>D:\buildtrees\dataframe\src\9b26a72fa1-4a13f75ffc.clean\src\Utils\DateTime.cc(988,9): error C3861: '_tzset': identifier not found [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>C:\Program Files (x86)\Windows Kits\10\bin\10.0.19041.0\XamlCompiler\Microsoft.Windows.UI.Xaml.Common.targets(486,5): error MSB4181: The "CompileXaml" task returned false but did not log an error. [D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj]
     4>Done Building Project "D:\buildtrees\dataframe\arm-uwp-dbg\DataFrame.vcxproj" (default targets) -- FAILED.
     3>Done Building Project "D:\buildtrees\dataframe\arm-uwp-dbg\ALL_BUILD.vcxproj" (default targets) -- FAILED.
     1>Done Building Project "D:\buildtrees\dataframe\arm-uwp-dbg\install.vcxproj" (default targets) -- FAILED.

Build FAILED.

```

--- a/ports/dataframe/CONTROL
+++ b/ports/dataframe/CONTROL
@@ -1,3 +1,4 @@
 Source: dataframe
 Version: 1.12.0
 Description: This is a C++ statistical library that provides an interface similar to Pandas package in Python
+Homepage: https://github.com/hosseinmoein/DataFrame

--- a/ports/dataframe/portfile.cmake
+++ b/ports/dataframe/portfile.cmake
@@ -8,7 +8,6 @@ vcpkg_from_github(
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
-    
     OPTIONS
         -DENABLE_TESTING:BOOL=OFF
     

--- a/ports/dataframe/portfile.cmake
+++ b/ports/dataframe/portfile.cmake
@@ -5,8 +5,6 @@ vcpkg_from_github(
     SHA512 43bd888312e16866f399e47adf7d0dc4fb4fd961063d3f53df4a69fb69f1ec95a1c294a4cefc4135f1acf6a4f27715fdcc71cb4bf69f2f72557290d9af439774
     HEAD_REF master
 )
-
-
 vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA

